hack/repos.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck disable=SC2034
+repos=(
+    api
+    apiextensions-apiserver
+    apimachinery
+    apiserver
+    cli-runtime
+    client-go
+    cloud-provider
+    cluster-bootstrap
+    code-generator
+    component-base
+    component-helpers
+    controller-manager
+    cri-api
+    csi-translation-lib
+    kube-aggregator
+    kube-controller-manager
+    kube-proxy
+    kube-scheduler
+    kubectl
+    kubelet
+    legacy-cloud-providers
+    metrics
+    mount-utils
+    pod-security-admission
+    sample-apiserver
+    sample-cli-plugin
+    sample-controller
```

Comment:
why don't we pull these from k/k?

Reply:
@sttts we could ... but this turns out to be a super set of all the repos that exist in all the current stable branches (gives that we may be introducing new repos and we may be moth-balling others). So a bit more complicated than `ls` in the correct directory under k/k master.